### ISSUE TITLE
Can't build&run sample, and can't use plugin

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [yanex/anko-template-project#11]
I tried to import this repo, but I got these errors when trying to build&run it:

![image](https://user-images.githubusercontent.com/5357526/30852459-8444da44-a2b4-11e7-8a98-53044e82ef08.png)

Also, I tried to install the plugin to preview the Anko layout ([this](https://plugins.jetbrains.com/plugin/7734-anko-support/update/19242)), but I don't see it...

How come?

**Comment by rcgonzalezf** [comment]
I'm getting the same issue with Android Studio 3.0, I was able to compile the project by commenting out the gravity line and leaving only one constructor:

` constructor(context: Context?) : super(context) {
        init()
    }`
Since it's just a demo, I think it could help you to have a glance, still it will be interesting to find a real solution to this problem and maybe do a PR with the fixed solution.

@AndroidDeveloperLB Do you still have the same issue?

**Comment by rcgonzalezf** [comment]
I was able to fix the compilation issue as you can see in the following PR:
https://github.com/Kotlin/anko-example/pull/13

TLDR:
Use the anko version '0.10.1'  instead of  '0.10.0-beta-2'

I hope it helps you.

**Comment by AndroidDeveloperLB** [comment]
Seems to work fine now.
Please update the repo with latest versions of everything. 
Android Studio reached RC version already.

I'm also pretty sure it will work with targetSdkVersion 26 .
